charts/prometheus-blackbox-exporter/values.yaml

```
@@ -213,8 +213,21 @@ extraArgs: []
 replicas: 1
 
 serviceMonitor:
-  ## If true, a ServiceMonitor CRD is created for a prometheus operator
-  ## https://github.com/coreos/prometheus-operator
+  ## If true, a ServiceMonitor CRD is created for a prometheus operator 
+  ## https://github.com/coreos/prometheus-operator for blackbox-exporter itself
+  ##
+  selfMonitor: false
```

i would prefer it like this:

```suggestion
  selfMonitor: 
    enabled: false
    additionalMetricsRelabels: {}
    additionalRelabeling: []
    labels: {}
    interval: 30s
    scrapeTimeout: 30s
```

---

Updated